Upgrade to Frictionless 5.5 (#51)
echemdb/echemdb: doc/news/frictionless5.rst

> **Changed:**
> 
> * Upgraded to frictionless >5.10.1.
> * Upgraded to svgdigitizer 0.8.0.
> * Changed doctest of `echemdb.local.collect_datapackages`, which now returns the first entry of a list of example datapackages.
> * Changed the 'echemdb' resource from 'type: buffer data' to 'type: pandas dataframe'.
> * Changed folder of examples generated with svgdigitizer.

Reviewer: ```suggestion
**Changed:**

* Changed the 'echemdb' resource from 'type: buffer data' to 'type: pandas dataframe'.

**Fixed:**

* Upgraded to work with frictionless >5.10.1.
* Upgraded to work with svgdigitizer 0.8.0.
* Fixed doctest of `echemdb.local.collect_datapackages`.

```

"Changed" is meant to talk about things that could potentially break stuff downstream. The change in doctesting does not qualify for that. Upgrading a dependency might break things but it will be obvious during install already and not be a subtle error at runtime. (Finallly, the directory change is not newsworthy I think ;) )

Author: ok :)
